eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -257,30 +258,30 @@ if ($Language -eq "javascript")
 
 if ($Language -eq "dotnet")
 {
-    $PublishedPkgs = Get-ChildItem "$($DocLocation)/packages" | Where-Object -FilterScript {$_.Name.EndsWith(".nupkg") -and -not $_.Name.EndsWith(".symbols.nupkg")}
-    $PublishedDocs = Get-ChildItem "$($DocLocation)" | Where-Object -FilterScript {$_.Name.StartsWith("Docs.")}
+    $PublishedPkgs = Get-ChildItem "$($DocLocation)" | Where-Object -FilterScript {$_.Name.EndsWith(".nupkg") -and -not $_.Name.EndsWith(".symbols.nupkg")}
+    $PublishedDocs = Get-ChildItem "$($DocLocation)" | Where-Object -FilterScript {$_.Name.EndsWith("docs.zip")}
 
-    foreach ($Item in $PublishedDocs) {
-        $PkgName = $Item.Name.Remove(0, 5)
-        $PkgFullName = $PublishedPkgs | Where-Object -FilterScript {$_.Name -match "$($PkgName).\d"}
-
-        if (($PkgFullName | Measure-Object).count -eq 1)
-        {
-            $DocVersion = $PkgFullName[0].BaseName.Remove(0, $PkgName.Length + 1)
-
-            Write-Host "Start Upload for $($PkgName)/$($DocVersion)"
-            Write-Host "DocDir $($Item)"
-            Write-Host "PkgName $($PkgName)"
-            Write-Host "DocVersion $($DocVersion)"
-            $releaseTag = RetrieveReleaseTag "Nuget" $PublicArtifactLocation 
-            Upload-Blobs -DocDir "$($Item)" -PkgName $PkgName -DocVersion $DocVersion -ReleaseTag $releaseTag
-        }
-        else
-        {
-            Write-Host "Package with the same name Exists. Upload Skipped"
-            continue
-        }
+    if (($PublishedPkgs.Count -gt 1) -or ($PublishedDoc.Count -gt 1))
+    {
+        Write-Host "$($DocLocation) contains more published artifacts than expected."
+        exit 1
     }
+    $pkgZip = $PublishedPkgs[0]
```

Comment:
Can we utilize VerifyPackage method in artifact-parsing-metadata.ps1? 
First, we can filter the nuget file (not symbol one) in folder "$BinDirectory/publicartifactscopy", then VerifyPackage is able to return us the right package metadata like
```
{ 
    $pkgName: "Azure.Core",
    $pkgVersion: ...
    $ReleaseTag: ...
    $ReadmeContent: ...
}
```

You don't need to parse $pkgName and $version again

Reply:
Also, there is no need to call RetrieveReleaseTag but to return the releaseTag from package metadata.

Reply:
Thanks @sima-zhu